packages/base/exports/index.ts

```diff
@@ -0,0 +1,23 @@
+import { ConstantsUtil } from '@web3modal/scaffold-utils'
+import { AppKit } from '../src/client.js'
+import type { AppKitOptions } from '../utils/TypesUtil.js'
+
+// -- Views ------------------------------------------------------------
+export * from '@web3modal/scaffold-ui'
+
+// -- Utils & Other -----------------------------------------------------
+export type * from '@web3modal/core'
+export { CoreHelperUtil } from '@web3modal/core'
+
+type CreateWeb3Modal = Omit<AppKitOptions, 'sdkType' | 'sdkVersion'>
+
+export function createWeb3Modal(options: CreateWeb3Modal) {
+  return new AppKit({
+    ...options,
+    sdkType: 'w3m',
+    sdkVersion: `html-multichain-${ConstantsUtil.VERSION}`
```

Comment:
Make sure to complete [SDK parameters refactor](https://www.notion.so/walletconnect/Multi-chain-SDK-parameters-refactor-for-Blockchain-API-endpoints-6b200e4f05e2465aa9e6785faef9ecdc?pvs=4) Notion doc and bring it to discussion again

Reply:
Not blocker for merging this, blocker when launching multichain